packages/zoe/test/unitTests/zcf/setupZcfTest.js

```diff
@@ -18,8 +18,14 @@ const contractRoot = `${dirname}/zcfTesterContract.js`;
 export const setupZCFTest = async (issuerKeywordRecord, terms) => {
   /** @type {ContractFacet} */
   let zcf;
+  /** @type {ContractFacet} */
+  let zcf2;
   const setZCF = jig => {
-    zcf = jig.zcf;
+    if (zcf === undefined) {
+      zcf = jig.zcf;
+    } else {
+      zcf2 = jig.zcf;
```

Comment:
It took me a while to verify this works as intended.

Would you mind adding a comment saying that these two branches are executed in separate contract vats, which is why `jig.zcf` refers to different objects in the two branches?

Reply:
Done!